step_definitions/course-steps.js

```diff
@@ -113,13 +122,13 @@ Then(/^.*clicks on save changes button$/, async function () {
 	await CRSSEditCopyCoursePage.clickSubmitButton();
 });
 
-Then(/^.*should see that course name (.*) with description correctly displayed (.*)$/, async function (courseName, description) {
+Then(/^.*course name (.*) with description correctly displayed (.*)$/, async function (courseName, description) {
 	let index = await courseListPage.getIndexOfGivenCourseInSection(courseName, courseListPage.section.activeCourses)+1;
 	let courseDescriptionAfterChanges = await courseListPage.getDescriptionCourse(index);
 	expect(description).to.equal(courseDescriptionAfterChanges);
 });
 
-Then(/^.*should see that course name (.*) with color correctly displayed (.*)$/, async function (courseName, colorCourse) {
+Then(/^.*course name (.*) with color correctly displayed (.*)$/, async function (courseName, colorCourse) {
```

Comment:
better: course name (.\*) with color (.\*) is displayed correctly

Reply:
My task concerns administration pages. I have only deleted what I thought was 100% redundant.